doc/api/api_changes_3.3/deprecations.rst

```diff
@@ -487,3 +487,13 @@ of all renderer classes is deprecated.
 
 `.transforms.AffineDeltaTransform` can be used as a replacement.  This API is
 experimental and may change in the future.
+
+``ColorbarBase`` parameters will become keyword-only
```

Comment:
This addition seems accidental.

Reply:
hmmm.  Happened during rebase.  Not sure why...